tests/unit/test_help_pages.py

```diff
@@ -100,7 +104,7 @@ def test_help_with_ops(self, capsys, mocked_config):
 
     def test_help_with_ops_with_plugins(self, capsys, mocker, mocked_config):
         mocker.patch(
-            "linodecli.arg_helpers.plugins.available",
+            "linodecli.help_pages.plugins.available",
```

Comment:
This fixes a unit test failure that was caused by the changed import [here](https://github.com/linode/linode-cli/pull/662/files#diff-e1a0bea9e101a8355b153028f34daf902221d2a1a4d1c812f85357a4ecd5633dR21)